.github/workflows/nodejs.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: ts integration
         run: yarn test-ts --selectProjects ts-integration
       - name: type tests
-        run: yarn tstyche --target 5.0,latest && yarn test-ts --selectProjects type-tests
+        run: yarn tstyche --target 5.0,current && yarn test-ts --selectProjects type-tests
```

Comment:
`current` also aligns with the strategy how `@tsd/typescript` is used at the moment: versions of `typescript` and `@tsd/typescript` were kept in sync. No need of this effort, because `current` simply uses the installed `typescript` package.